Documentation/op-guide/monitoring.md

```diff
@@ -5,7 +5,8 @@ Each etcd server exports metrics under the `/metrics` path on its client port.
 The metrics can be fetched with `curl`:
 
 ```sh
-$ curl -L http://localhost:2379/metrics | grep -v debugging # ignore unstable debugging metrics
+# OR http://localhost:9379/metrics when etcd is run with '--expose-metrics' flag
+$ curl -L http://localhost:9379/metrics | grep -v debugging # ignore unstable debugging metrics
```

Comment:
Should stick to the default 2379 here since if the user is changing the ports, they probably don't need this guide any more. Instead of hiding an explanation in a comment, the doc can directly say that optionally metrics can be exported to other listen addresses with `--listen-metrics-urls` where it's talking about the client port